app/views/registrations/school_profile/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% content_for :before_content, govuk_back_link(
+      text: "Back",
+      href: registrations_question_two_path)
+    %>
+
+    <h2 class="govuk-heading-l">What is your school Unique Reference Number(URN)?</h2>
+
+    <%= form_for(:school, model: @school_profile_form, url: registrations_school_profile_path) do |f| %>
+      <label class="govuk-hint">
+        This is a 6 digit number. Not sure? <a target='_blank' href='https://www.get-information-schools.service.gov.uk/'>Search for your school</a> on our register.
+        <%= f.govuk_text_field(:urn, width: 20, label:  nil) %>
+      </label>
+      <%= f.govuk_submit "Continue" %>
+    <% end %>
```

Comment:
Just an idea that you might want to relay to Al (if it isn't in the works already) - when we have the component for school search that Sebastian was working on, it might be a lot easier to use that for finding the URN.

Reply:
Yes, he's got a ticket for it already and the plan is to use it there